lib/tasks/openapi.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'open_api'
+
+desc 'Generate openapi V3 documentation'
+namespace :documentation do
+  task :generate => :environment do
+    OpenApi.write_docs
+  end
+end
+
+task :test => 'documentation:generate'
```

Comment:
How about we drop this from here and move doc generation before `test:validate` in `.travis.yml`?

Reply:
`rake test` won't run without generated documentation since integration tests use generated documentation to validate requests and responses against the schema.